tests/system-fetcher/tests/handler_test.go

```diff
@@ -97,12 +98,14 @@ func TestSystemFetcherSuccess(t *testing.T) {
 	setMockSystems(t, mockSystems, tenant.TestTenants.GetDefaultTenantID())
 	defer cleanupMockSystems(t)
 
-	template, err := fixtures.CreateApplicationTemplateFromInput(t, ctx, certSecuredGraphQLClient, tenant.TestTenants.GetDefaultTenantID(), fixApplicationTemplate("temp1"))
+	appTemplateName1 := fmt.Sprintf("SAP %s", "temp1")
```

Comment:
Extract this into a function like `createAppTemplateName` in the director e2e tests.